### PR TITLE
Second attempt to use class fixtures in TestCase (bug 1142943)

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

**Comment by diox** [comment]
r+

File: mkt/websites/indexers.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def get_mapping(cls):
                 'properties': {
                     'id': {'type': 'long'},
                     'category': cls.string_not_analyzed(),
-                    'created': {'type': 'date', 'format': 'dateOptionalTime'},
+                    'created': {'format': 'dateOptionalTime'},
```
Comment (robhudson) [inline mkt/websites/indexers.py]: Is this supposed to be here?

**Comment by robhudson** [comment]
Looks like some infinite recursion happening in the test suite?

File: mkt/websites/indexers.py (hunk 1 above)
Comment (diox) [inline mkt/websites/indexers.py]: Yeah, it was added on purpose to reproduce the issue that caused us to revert this patch the previous time. If you look at the travis build, you'll see a failure caused by this particular change, which is fine, and then a bunch of other failures that should not happen. The goal is to find out what causes those errors and then when we do, we can remove this part of the patch.

**Comment by eviljeff** [comment]
for reference: see https://github.com/mozilla/zamboni/pull/2979 and https://github.com/mozilla/zamboni/pull/3001 for previous pull and revert

**Comment by eviljeff** [comment]
The tests are failing as expected now, after moving some of the initialisation in ESTestCase to setUpTestData from setUpClass and putting both fixture and test data setup in a try.  Just need to fix the (purposely) broken code in indexers.py before merging.

**Comment by diox** [comment]
r+

**Pull request merged**